nosql/1.0/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Jakarta NoSQL 1.0 (under development)"
+date: 2020-07-23
+summary: "Under Development"
+---
+
+Jakarta NoSQL is a Java API standard that streamlines the integration of Java applications with NoSQL databases. It defines a set of APIs and provides a standard implementation for most NoSQL databases. 
+The goal is to create the specification in Jakarta EE to help Jakarta EE developers create enterprise-grade applications using Java® and NoSQL technologies. It helps them create scalable applications while maintaining low coupling with the underlying NoSQL technology.
+
+In general, we made some progress, especially with the creation of TCKs:
+https://github.com/eclipse-ee4j/nosql/tree/master/tck
+All APIs proved to be quite stable, except for the Graph API, and that is why there is no API for graphs. We are waiting for the definition of a standard graphical communication API, either [Apache TinkerPop](http://tinkerpop.apache.org/) or [Open Cypher](https://www.opencypher.org/), led by [Neo4J](https://neo4j.com/). Therefore, we only have a Graph implementation on the RI side, but not in the API, at least for a while.
```

Comment:
Jakarta has no concept of RI (aka Reference Implementations), AFAIK. Should be either changed to implementation project name or made clear that RI is the project name

Reply:
Indeed, we use compatible implementation as far as I know

Reply:
I'd say "implementation side" here instead.